feg/radius/src/modules/module_test_utils.go

```diff
@@ -39,6 +40,7 @@ func WaitForRadiusServerToBeReady(secret []byte, addr string) (err error) {
 		if err == nil {
 			return nil
 		}
+		time.Sleep(time.Millisecond)
```

Comment:
Well, I am always told adding sleeps in any way is bad practice (even in retry loops like this), so I would suggest asking others if they're fine with it as well. 
Personally, I think in a retry loop to wait a short time before checking again makes sense, especially if it is just a millisecond and easily fixes all the flakiness of the tests. (alternatively one could increase the number of retries, which, in essence, does the same thing).

Reply:
I think we should have a bit more than a millisecond since the necessary wait time might depend on the machine this is running on?

Reply:
Thanks for the comments! I discussed this briefly with @sebathomas and we thought a short sleep was a reasonable solution, but I take your point - I was also debating whether to just increase the number of retries but thought that a short sleep was a little nicer. I don't have a strong opinion on this though. Regarding the length of the sleep, for reference, the times to run the tests in the relevant folders on this branch are:

modules/coafixedip -> Local: 15ms, CI: 7ms
modules/coanas -> Local: 10ms, CI: 23ms
modules/lbserve -> Local: 8ms, CI: 9ms

so I thought a millisecond was in the right ball park, but perhaps something a little higher is better. 5ms perhaps?

Reply:
I actually prefer having a sleep here because it defines a minimum time scale for the waiting. Otherwise how long we wait just depends on how long it takes to check if the server is running.

And if it works with a millisecond sleep, I don't really see a disadvantage. You could increase it slightly or increase the number of retries to make it more stable.

(The problem is that we need the busy waiting at all, and in production code it might make sense to avoid even such short sleeps, but for tests I don't see the problem.)

Reply:
I've increased the sleep a little. Let me know what you think @MoritzThomasHuebner